tests/ttnn/integration_tests/squeezebert/test_ttnn_squeezebert.py

```diff
@@ -0,0 +1,364 @@
+# SPDX-FileCopyrightText: © 2023 Tenstorrent Inc.
+
+# SPDX-License-Identifier: Apache-2.0
+
+import ttnn
+import torch
+import pytest
+import transformers
+
+from tests.ttnn.utils_for_testing import assert_with_pcc
+from ttnn.model_preprocessing import preprocess_model_parameters
+from models.utility_functions import skip_for_wormhole_b0, torch_random
+from models.experimental.functional_squeezebert.tt import ttnn_functional_squeezebert
+
+from transformers import SqueezeBertForQuestionAnswering
+
+
+@skip_for_wormhole_b0()
+@pytest.mark.parametrize("model_name", ["squeezebert/squeezebert-uncased"])
+@pytest.mark.parametrize("batch_size", [8])
```

Comment:
Why is there a `test_ttnn_squeezebert.py` in here and in `models/experimental`? Why do we need so many?

Reply:
Any action on this?

Reply:
The test is removed from `models/experimental/` and is now only in `tests/ttnn/integration_tests/`.
Corresponding commit - [link](https://github.com/tenstorrent/tt-metal/compare/main...keerthana/functional_squeezebert_). 
Thank you.